pyccel/ast/core.py

```diff
@@ -74,10 +74,11 @@
     'If',
     'IfSection',
     'Import',
-    'InlineFunctionDef',
     'InProgram',
+    'InlineFunctionDef',
```

Comment:
Any idea why this was moved? The others seem to be alphabetical, but not here

Reply:
all the moves should be alphabetical as far as i know i think the `P` < `l` in `Program` and `line`

Reply:
Ah! That explains it. It's not alphabetical but it is ordered by characters:+1: